README.md

```diff
@@ -1 +1,107 @@
-# appointment-diary-front-end
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+
+# Book Vehicle
+
+> The Book Vehicle is a a React, Redux application based on the [back-end API-docs](https://book-vehicle.herokuapp.com/).
+
+- In this application we're fetching the returned data from the API in the Redux store.
+
+- The user can reserve a vehicle and it'll update the state automatically, , and also the user can add a vehicle in this application.
+
+## Built With
+
+- React
+- Redux
+- React Bootstrap
+- HTML
+- CSS
+- Javascript
+- node-modules
+- lighthouse
+- hintweb
+- stylint
+- eslint
+- webpack
+- web server
+- plugins for HTML and CSS
+
+## Live Demo
+
+- [Live Demo Link]()
+
+## Link to Back-end ripo

```

Comment:
- [ ] This line has a typo.  It should be "repo" instead of "ripo".  Kindly fix this spelling mistake.